### PR TITLE
Add support for Gulp file with Babel

### PR DESCRIPTION
**Pull request by mamadoo**
**Changes proposed:**
* Add

**Things I've done:**
* Add support for gulpfile.babel.js

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -158,6 +158,7 @@ export const extensions: IFileCollection = {
     { icon: 'groovy2', extensions: [], languages: [languages.groovy], format: FileFormat.svg, disabled: true },
     { icon: 'grunt', extensions: ['gruntfile.js'], filename: true, format: FileFormat.svg },
     { icon: 'gulp', extensions: ['gulpfile.js'], filename: true, format: FileFormat.svg },
+    { icon: 'gulp', extensions: ['gulpfile.babel.js'], filename: true, format: FileFormat.svg },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: do not redefine the extension, just add it the line above like this:
`{ icon: 'gulp', extensions: ['gulpfile.js', 'gulpfile.babel.js'], filename: true, format: FileFormat.svg },`

Comment (mamadoo) [inline src/icon-manifest/supportedExtensions.ts]: ok, got it.
Thanks.

**Comment by robertohuertasm** [comment]
@mamadoo could you rebase your branch with our master? I'll merge it after that. Thanks for your PR!

**Comment by jens1o** [comment]
We can do that too @robertohuertasm ?

**Comment by robertohuertasm** [comment]
Yes, I know but I prefer users do that if possible.

**Comment by jens1o** [comment]
Okay.

**Comment by mamadoo** [comment]
I don't clone the forked repository in my computer at first. I change this file directly in github online and the create the PR for that. I don't know how to rebase forked account directly in github online (i just know in command line).
If you can Please do it.
If not i clone in computer and then pull it.

**Comment by JimiC** [comment]
@mamadoo What you did it equally fine.

**Pull request merged**